api.swagger.yaml

```diff
@@ -465,7 +465,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/uuid-list-schema'
+                $ref: '#/components/schemas/location-uuid-schema'
```

Comment:
Is it intentional that we are also changing from a list of locations to a single location?